components/salesforce_rest_api/actions/create-contact/create-contact.mjs

```diff
@@ -0,0 +1,48 @@
+import salesforce from "../../salesforce_rest_api.app.mjs";
+import contact from "../../common/sobjects/contact.mjs";
+import lodash from "lodash";
+import { toSingleLineString } from "../../common/utils.mjs";
+
+export default {
+  key: "salesforce_rest_api-create-contact",
+  name: "Create Contact",
+  description: toSingleLineString(`
+    Creates a Contact, which is a person associated with an account.
+    See [Contact SObject](https://developer.salesforce.com/docs/atlas.en-us.228.0.object_reference.meta/object_reference/sforce_api_objects_contact.htm)
+    and [Create Record](https://developer.salesforce.com/docs/atlas.en-us.228.0.api_rest.meta/api_rest/dome_sobject_create.htm)
+  `),
+  version: "0.2.2",
+  type: "action",
+  props: {
+    salesforce,
+    LastName: {
+      type: "string",
+      label: "LastName",
+      description: "Required. Last name of the contact up to 80 characters.",
```

Comment:
Can we remove the required and add some internal validation for the 80 chars limit?
```suggestion
      description: "Last name of the contact up to 80 characters.",
```
Also, maybe we can move this to the app? I saw this being used in some other files